ui/shared/components/panel/family/FamilyReads.jsx

```diff
@@ -367,19 +390,29 @@ class FamilyReads extends React.PureComponent {
                 )}
               </div>
             )}
+            { locusOptions && (
```

Comment:
this should display above the rna seq tracks/filters

Reply:
Don't you think the rna seq tracks should stay together with the DNA tracks?

Reply:
no, like I said, I think the order should be: select track type, select range, select any rna-specific stuff